src/math_parser_diag.cpp

```diff
@@ -871,6 +874,9 @@ bool _filter_monster( Creature const &critter, std::vector<ID> const &ids, int r
     return false;
 }
 
+// TODO: some form of notation or sentinel value for referencing
```

Comment:
Is this about the `kwargs.kwarg_or( "radius", 1000 );` below?

Can we do PICKUP_RANGE * 10? That is 600 right now.

I am asking because I don't get it.

Reply:
It's for `monsters_nearby()` math field